test/e2e/crud.go

```diff
@@ -23,10 +23,22 @@ import (
 
 	kubeclientset "k8s.io/client-go/kubernetes"
 	"k8s.io/federation/pkg/federatedtypes"
+	"k8s.io/federation/test/common"
 	fedframework "k8s.io/federation/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+var _ = framework.KubeDescribe("Federated namespace [Feature:Federation][Experimental]", func() {
+	f := fedframework.NewDefaultFederatedFramework("federated-namespace")
+	Describe(fmt.Sprintf("Federated namespace resources"), func() {
+		It("should create, read, update and delete successfully", func() {
```

Comment:
Consider updating this test description to reflect the nature of this test (can likely cribbed from the test that is being replaced).